clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/provider/agent/DockerRegistryImageCachingAgent.groovy

```diff
@@ -157,11 +158,17 @@ class DockerRegistryImageCachingAgent implements CachingAgent, AccountAware, Age
             }
           }
         }
+        try {
+          creationDate = credentials.client.getCreationDate(repository, tag)
+        } catch (Exception e) {
+          log.warn("Unable to fetch tag creation date, reason: {} (tag: {}, repository: {})", e.message, tag, repository)
+        }
 
         cachedTags[tagKey].with {
           attributes.name = "${repository}:${tag}".toString()
           attributes.account = accountName
           attributes.digest = digest
+          attributes.creationDate = creationDate
```

Comment:
Would you mind changing this to `attributes.date`? We have an alternate docker tag caching agent internally @ Netflix that store a `date` attribute used for sorting via an alternate controller, just like in this PR. It reads from ElasticSearch instead of an actual registry which is why it's currently internal-only, but we do plan on merging its functionality into `DockerRegistryImageCachingAgent` at some point, at which point naming consistency will help.

Reply:
Hi @asher 
Thank you for the comment.

Sure! I've changed it to `attributes.date`. Please have a look at it.
And good to know you have a plan to merge Netflix internal functionality in the future 👍